oaknational/Oak-Web-Application
# [pupil-765] Remove 404 for isSensitive lessons as there is now a modal warning.

## Description

This removes the 404 page from lessons marked with isSensitive. 

For info, there is already a modal that warns users that there is sensitive content.

In addition these lessons are only discoverable through the teacher experience and are not index-able by search engines.


## How to test

1. Go to https://deploy-preview-2506--oak-web-application.netlify.thenational.academy/pupils/lessons/sexual-consent-and-the-law-6gt3et/overview
2. You should see a lesson with a modal warning

_

1. Check that there is a `<meta name="robots" content="noindex,nofollow">` within the head of the document.

## Checklist
